File: Python/Avg_without_array.py
#Average of N numbers
print("Enter number of elements : ",end='')
n=int(input())
print("Enter elements : ",end='')
temp=0
total=0
while(temp!=n):
    val=int(input())
    total=total+val
    temp=temp+1
avg=total/n;
print("Average :",avg)

File: C++/Avg_without_array.cpp
//Average of N numbers
#include<iostream>
using namespace std;
int main(){
    int n,temp,val,sum;
    float avg;
    cout<<"Enter number of elements : ";
    cin>>n;
    cout<<"Enter elements :";
    temp=0;
    sum=0;
    while(temp!=n){
        cin>>val;
        sum=sum+val;
        temp++;
    }
    avg=sum/n;
    cout<<"Average : "<<avg;
    return 0;
}
